build/assert-publish-aot-testApp-warning-count.ps1

```diff
@@ -12,6 +12,20 @@ foreach ($line in $($publishOutput -split "`r`n"))
     }
 }
 
+pushd $rootDirectory/test/OpenTelemetry.AotCompatibility.TestApp/bin/Debug/net7.0/linux-x64

```

Comment:
Be consistent with forward slash here since this test is only being run on Linux?

Update the command for dotnet publish:
`dotnet publish $rootDirectory\test\OpenTelemetry.AotCompatibility.TestApp\OpenTelemetry.AotCompatibility.TestApp.csproj -nodeReuse:false /p:UseSharedCompilation=false`

Reply:
updated!